pkgs/applications/virtualization/conmon/default.nix

```diff
@@ -0,0 +1,39 @@
+{ flavor ? ""
+, ldflags ? ""
+, stdenv
+, fetchFromGitHub
+, git
+, pkgconfig
+, glib
+, glibc
+, systemd
+}:
+
+stdenv.mkDerivation rec {
+  project = "conmon";
+  name = "${project}-${version}${flavor}";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "containers";
+    repo = project;
+    rev = "v${version}";
+    sha256 = "08fgkbv7hq62lcq39za9pm2s2j92ismgrkvfm7acwbvajqh9syjb";
+  };
+
+  nativeBuildInputs = [ git pkgconfig ];
```

Comment:
Why is `git` now necessary? `fetchFromGitHub` & co do not need it at all and won't include the `.git` subdirectory anyway.

If there is some usage of git in the build system (e.g. running `git rev-parse HEAD` and checking `$?` to see if you're in a repo or not), would it be feasible to instead just include a `patchPhase` that instead fixed this to *not* use `git`?

Reply:
Yeah its the second reason. I excluded git since it won't be needed in further versions because of an upstream fix of within the build process.